playbooks/centurion_teams.yaml

```diff
@@ -0,0 +1,244 @@
+- name: Centurion ERP Teams Setup
```

Comment:
the file names need to be fixed too

`inventory` and `teams` is sufficient.

Reply:
Ack, will make that change